info.xml

```diff
@@ -14,7 +14,7 @@
     <url desc="Support">https://github.com/systopia/de.systopia.xcm/issues</url>
     <url desc="Licensing">https://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
-  <releaseDate></releaseDate>
```

Comment:
I don't like *Civix* re-formatting the `<releaseDate>` element as a self-closing tag, as with every released version, this will change the tag format and not just the date value. I always reset that *Civix* change manually … sorry for being picky :nerd_face:

Reply:
let me know if you want me to fix it or you can push the change